bark_ml/environments/gym.py

```diff
@@ -28,11 +28,10 @@ def __init__(self):
       blueprint=cont_highway_bp, render=True)
 
 class DiscreteHighwayGym(SingleAgentRuntime, gym.Env):
-  def __init__(self):
-    params = ParameterServer()
+  def __init__(self, params = ParameterServer(), render=False):
```

Comment:
why false as default. this is not as before

Reply:
As per gym environment convention,  render is explicitly called by env.render() when needed, where as keeping the render=True causes the environment to be rendered in the env.step() instead. Of course it can still be achieved by passing render=True.

Reply:
makes sense 👍